src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,35 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private static final DateTimeFormatter PATTERN = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        StringBuilder strB = new StringBuilder();
```

Comment:
do not use such naming, how to guess what it stores?
```suggestion
        StringBuilder report= new StringBuilder();
```